sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/csv/UnivocityParser.scala

```diff
@@ -378,12 +378,15 @@ private[sql] object UnivocityParser {
   def tokenizeStream(
       inputStream: InputStream,
       shouldDropHeader: Boolean,
+      skipLines: Int,
       tokenizer: CsvParser,
       encoding: String): Iterator[Array[String]] = {
+    val handleSkipLines: () => Unit =
+      () => 1.to(skipLines).foreach(_ => tokenizer.parseNext())
```

Comment:
Whats the behaviour when skipLines is greater than the length of the input file?